src/app/components/Header.jsx

```diff
@@ -0,0 +1,14 @@
+export function Header({ appMode }) {
+  return (
+    <div className="flex justify-between uppercase">
+      <div className="flex flex-col text-sm">
+        <span>Sudoku</span>
+        <span>Cruncher</span> 
+      </div>
+      <div className="flex items-center">
+        <div className={`p-2 text-[14px] border-r-2 border-black ${appMode === "creator" && "text-gray-400"}`}>Solver</div> 
+        <div className={`p-2 text-[14px] ${appMode === "solver" && "text-gray-400"}`}>Creator</div> 
```

Comment:
It's not that crucial but it's better to use text-sm instead of specifying the pixels to save memory. To be fair it really doesn't matter at this scale but I'm not sure if I'm supposed to pass this PR

Reply:
Thats actually good catch font-sm is 14px anyway by default I thought it's 12. I'll change it. I'm also curious how does usage of exact px size increase memory consumption?